tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -327,7 +327,6 @@ public function testGetAccountLogs($data): array
         $this->assertEquals('desktop', $response['body']['logs'][2]['deviceName']);
         $this->assertEquals('', $response['body']['logs'][2]['deviceBrand']);
         $this->assertEquals('', $response['body']['logs'][2]['deviceModel']);
-        $this->assertEquals($response['body']['logs'][2]['ip'], filter_var($response['body']['logs'][2]['ip'], FILTER_VALIDATE_IP));
```

Comment:
How come this was removed?

Reply:
We already do the assertions check [here](https://github.com/appwrite/appwrite/pull/8236/files/485b8eeb8b2461f46a877cbc017baeaf487a9c33#diff-434428e8237205f1c4c6540329c60816d1e243383b6a7b044b844a594f4a35edL296) . Or its different? 👀